lib/firmware_model.dart

```diff
@@ -50,6 +50,9 @@ class FirmwareModel extends SafeChangeNotifier {
   Future<void> install(FwupdDevice device, FwupdRelease release) async {
     try {
       await _service.install(device, release);
+      if (device.flags.contains(FwupdDeviceFlag.needsReboot)) {
```

Comment:
this is too drastic action to take automatically. we need to show a confirmation dialog and ask the user :)